src/backend/export/outputVendors/googleSheets/googleSheets.ts

```diff
@@ -11,7 +11,11 @@ import { appendToSpreadsheet } from './googleSheetsInternalAPI';
 
 const GOOGLE_SHEETS_DATE_FORMAT = 'DD/MM/YYYY';
 const DEFAULT_SHEET_NAME = '_caspion';
-const COLUMN_HEADERS = ['תאריך', 'סכום', 'תיאור', 'תיאור נוסף', 'קטגוריה', 'מספר חשבון', 'hash - לא לגעת', 'מטבע'];
+const COLUMN_HEADERS = [
```

Comment:
It looks like the order of the columns changed. This will mean that the sheets that will be created going forward will not have the correct header order.
Please retain the same order and add the new columns to the end (also making sure they are the same order as the code you added in transactionsToCreate